carbon-client/carbon

```diff
@@ -206,6 +85,17 @@ debug_specs() {
     fi
 }
 
+write_payload() { 
+    PAYLOAD_FILE="$LOG_DIR/payload-$UUID.json"
+    echo "$PAYLOAD," >> $PAYLOAD_FILE
+    if [ -z $QUIET ] ; then
+        echo "This command has been set to offline mode or has been unable to reach the Internet"
+        echo "so is unable to send system specs to the carbon leaderboard."
+        echo
+        echo "Upload $PAYLOAD_FILE to $LEADERBOARD_URL to get your carbon calculations."
```

Comment:
Could change this to `$LEADERBOARD_URL/data-entry` to point users directly to the new data entry page.